src/chi/themes/brightspeed/_variables.scss

```
@@ -1316,6 +1316,22 @@ $range-slider-disabled-opacity:             $opacity-60;
 $range-slider-focus-color:                  $focus-color;
 $range-slider-tick-border-color:            $color-border-base-dark;
 
+// Save view
+$save-view-background-color:                $color-yellow-30;
+$save-view-border-color:                    transparent;
+$save-view-divider-border-color:            rgba($color-black, 0.5);
+$save-view-divider-height:                  1.25rem;
+$save-view-results-font-size:               $font-size-md;
+$save-view-label-font-size:                 $font-size-md;
+$save-view-label-font-weight:               $font-weight-semi-bold;
+$save-view-save-border-color:               $color-border-info;
+$save-view-save-border-style:               dashed;
+$save-view-ribbon-background-color:         $color-blue-70;
+$save-view-ribbon-label-color:              $color-text-white;
+$save-view-ribbon-label-font-style:         italic;
+$save-view-ribbon-arrow-border-color:       $color-blue-70;
```

We can't render the Lumen blue color in the Brightspeed theme. I would recommend using black.

```suggestion
$save-view-ribbon-background-color:         $color-black;
$save-view-ribbon-label-color:              $color-text-white;
$save-view-ribbon-label-font-style:         italic;
$save-view-ribbon-arrow-border-color:       $color-black;
```